stars-core/src/main/kotlin/tools/aqua/stars/core/metric/metrics/postEvaluation/MissedPredicateCombinationsPerTSCMetric.kt

```diff
@@ -47,15 +50,18 @@ import tools.aqua.stars.core.types.*
  * @property logger [Logger] instance.
  */
 @Suppress("unused")
-class MissingPredicateCombinationsPerTSCMetric<
+class MissedPredicateCombinationsPerTSCMetric<
     E : EntityType<E, T, S, U, D>,
     T : TickDataType<E, T, S, U, D>,
     S : SegmentType<E, T, S, U, D>,
     U : TickUnit<U, D>,
     D : TickDifference<D>>(
     override val dependsOn: ValidTSCInstancesPerTSCMetric<E, T, S, U, D>,
-    override val logger: Logger = Loggable.getLogger("missing-predicate-combinations")
-) : PostEvaluationMetricProvider<E, T, S, U, D>, Loggable {
+    override val logger: Logger = Loggable.getLogger("missed-predicate-combinations")
+) : PostEvaluationMetricProvider<E, T, S, U, D>, Serializable, Loggable {
+
+  /** Holds the evaluation result after calling [postEvaluate]. */
+  private var evaluationResult: Map<TSC<E, T, S, U, D>, Set<PredicateCombination>>? = null
```

Comment:
Consider renaming to evaluationResultCache

Reply:
Resolved in 018a5bb4836705e27b19d0e43df1ff2ee0917457